lib/install/validate/validator.go

```diff
@@ -205,6 +205,20 @@ func (v *Validator) basics(ctx context.Context, input *data.Data, conf *config.V
 	conf.SetName(input.DisplayName)
 	conf.SetDebug(input.Debug.Debug)
 	conf.Name = input.DisplayName
+
+	// Set default size of scratch disk
+	if input.ScratchSize == "" {
```

Comment:
yeah I wasn't 100% sure where this should go (a common problem with me) so I was expecting this feedback thank you @emlin